docs/references/uvh5_memo.tex

```diff
@@ -399,16 +468,51 @@ \section{Version History}
 writing fully compatible readers/writers may look there for further details.
 
 It is strongly encouraged that independent UVH5 writers conform to the latest
-version (Version 1.0 at time of writing), while readers are encouraged to
+version (Version 1.1 at time of writing), while readers are encouraged to
 support backwards compatibility as much as possible. If readers cannot support
 all revisions, reading more recent versions should be prioritized.
 
+
 \subsection{\texttt{version} dataset}
 When present, the version information is stored in the Header as a string-based
 dataset with the key \verb+version+. Note that files have not always contained
 this dataset, but as much as possible, new files written should contain this
 dataset to clarify.
 
+\subsection{Version 1.0}
```

Comment:
This subsection has the same name as a later subsection. It clearly needs to be renamed, and maybe it should move later (after the 0.x/0.1 subsection)?

Reply:
@plaplant Can you recommend an ordering here? Should it be chronological? And should this subsection should be called 1.1?

Reply:
I think I fixed this to be more sensible now. Of course it still needs review!